data/migrations/0039_add_input_file_sequences.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-12-12 16:49
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+def populate_sequence_fields(apps, schema_editor):
+    populate_sequence_for_model(apps.get_model('data', 'DDSJobInputFile'))
+    populate_sequence_for_model(apps.get_model('data', 'URLJobInputFile'))
+
+
+def populate_sequence_for_model(model):
+    # Fill in sequence_group and sequence with numbers so we can get a baseline of sorting.
+    for input_file in model.objects.all():
+        input_file.sequence_group = input_file.id
+        input_file.sequence = 0  # first item in this sequence group
```

Comment:
For migrating existing data where there's only one group of files (e.g. sample pairs), wouldn't it be more accurate to use a default `sequence_group` but variable `sequence`?

We discussed future UI where we'd partition stage groups by their `sequence_group`. This default would create a bunch of groups of size 1